static/app/views/explore/logs/logsAutoRefresh.tsx

```diff
@@ -127,18 +137,28 @@ function getPreFlightDisableReason({
   sortBys,
   hasAbsoluteDates,
   mode,
+  visualizes,
   averageLogsPerSecond,
   initialIsError,
 }: {
   hasAbsoluteDates: boolean;
   mode: Mode;
   sortBys: ReturnType<typeof useLogsSortBys>;
+  visualizes: readonly Visualize[];
   averageLogsPerSecond?: number | null;
   initialIsError?: boolean;
 }): PreFlightDisableReason | null {
   if (mode === Mode.AGGREGATE) {
     return 'aggregates';
   }
+  if (
+    visualizes.some(
+      visualize =>
+        visualize.yAxis !== `${AggregationKey.COUNT}(${OurLogKnownFieldKey.MESSAGE})`
```

Comment:
I believe in the newer typescript you can enforce `${AggregationKey}(${OurLogFieldKey})`